GenericsHomework.Tests/NodeTests.cs

```diff
@@ -0,0 +1,115 @@
+using Xunit;
+
+namespace GenericsHomework.Tests;
+
+public class NodeTests
+{
+    [Fact]
+    public void Next_NodeLoopsOnItself_Equal()
+    {
+        Node<int> node = new(3);
+        Assert.Equal(node.Data, node.Next.Data);
+    }
+
+    [Fact]
+    public void Append_Node_Equal()
+    {
+        Node<int> node = new(6);
+        node.Append(7);
+        Assert.Equal(7, node.Next.Data);
+    }
+
+    [Fact]
+    public void Append_NodeWithDifValue_NotEqual()
```

Comment:
```suggestion
    public void Append_NodeWithDifferentValue_NotEqual()
```
No need to use shorthand